easybuild/tools/github.py

```diff
@@ -324,6 +326,14 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    os.chdir(develop)
+    patch_name = 'patch%s' % pr
+    patch_url = URL_SEPARATOR.join([GITHUB_URL, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, GITHUB_PR, str(pr)])
+    patch = download_file(patch_name, patch_url + '.patch', path + '/' + patch_name)
+
+    result = apply_patch(patch, develop, level=1)
```

Comment:
also, check `result` to make sure the patch applied successfully?

it's not unlikely this will fail, e.g. when there's a merge conflict in the PR that needs to be resolved